File: app/src/main/java/ru/otvykaniye/tracker/SmallTrackerWidgetProvider.java
package ru.otvykaniye.tracker;

import android.appwidget.AppWidgetManager;
import android.appwidget.AppWidgetProvider;
import android.content.Context;

public class SmallTrackerWidgetProvider extends AppWidgetProvider {
    @Override public void onUpdate(Context context, AppWidgetManager manager, int[] ids) {
        manager.updateAppWidget(ids, WidgetUi.small(context));
    }
    @Override public void onEnabled(Context context) { WidgetUi.updateAll(context); }
}

File: app/src/main/java/ru/otvykaniye/tracker/SmallTrackerWidgetProvider.kt
package ru.otvykaniye.tracker

import android.appwidget.AppWidgetManager
import android.appwidget.AppWidgetProvider
import android.content.Context

class SmallTrackerWidgetProvider : AppWidgetProvider() {
    override fun onUpdate(context: Context, manager: AppWidgetManager, ids: IntArray) {
        manager.updateAppWidget(ids, WidgetUi.small(context))
    }
    override fun onEnabled(context: Context) {
        WidgetUi.updateAll(context)
    }
}
